README.md

```diff
@@ -2,19 +2,29 @@
 
 Radar Software Toolkit
 ========
+The Radar Software Toolkit (RST) is maintained by the SuperDARN Data Analysis Working Group (DAWG). To find out what we are currently working on, take a look at our [projects](https://github.com/SuperDARN/rst/projects) page, and also the current [issues](https://github.com/SuperDARN/rst/issues) and [pull requests](https://github.com/SuperDARN/rst/pulls). 
 
+## Documentation
 RST's documentation is currently hosted on two sites:
 - RST readthedocs includes the installation guide, RST Tutorials, and SuperDARN data formats (coming soon):
   https://radar-software-toolkit-rst.readthedocs.io/en/latest/
 - RST API documentation includes the software structure and binary command line description and options: 
   https://superdarn.github.io/rst/
 
-## Getting Started 
+## Installation
 
 Installation guide for:
   - [Linux](https://radar-software-toolkit-rst.readthedocs.io/en/latest/user_guide/linux_install/)
   - [MacOSX](https://radar-software-toolkit-rst.readthedocs.io/en/latest/user_guide/mac_install/)
   - <font color="grey">Windows </font> yet to be implemented
+  
+## Contribute to RST
+The DAWG welcomes new testers and developers to join the team. Here are some ways to contribute:
+
+  - Test pull requests: to determine which [pull requests](https://github.com/SuperDARN/rst/pulls) need to be tested right away, filter them by their milestones
+  - Get involved in projects: the [projects](https://github.com/SuperDARN/rst/projects) page is used to organize our priority areas. Each project board shows items which are in progress, ready for testing/review, and completed.
+ - Discuss [issues](https://github.com/SuperDARN/rst/issues) and answer questions
+ - Become a developer: if you would like to contribute code to RST, please contact the DAWG chairs: Kevin Sterne (kevintyler *at* vt *dot* edu) and Emma Bland (emmab *at* unis *dot* no)
```

Comment:
Not sure where we landed with this. Should we replace the email addresses with the mailing list one?
```suggestion
 - Become a developer: if you would like to contribute code to RST, please contact us at darn-dawg@isee.nagoya-u.ac.jp or submit a new [issue](https://github.com/SuperDARN/rst/issues) on Github.
```

Reply:
I thought we were fine with them contacting us directly. I'm almost certain we tested out that you don't have to be on a list to send an e-mail to it.  I do want to change this some though.

Reply:
@ksterne Really just for future-proofing, or are we doomed to be the DAWG chairs forever?! 🤣

Reply:
CHAIRS FOREVER!  Good point on keeping it future-proofed though.  One less thing that's hardcoded in.